docs/self-managed/concepts/multi-region/dual-region.md

```diff
@@ -196,12 +190,12 @@ The **Recovery Time Objective (RTO)** is the time to restore services to a funct
 
 For Zeebe the **RPO** is **0**.
 
-For Operate and Tasklist the **RPO** is close to **0** for critical data due to the previously mentioned small chance of data loss in Elasticsearch during the failback procedure.
+For Operate and Tasklist the **RPO** is **0**.
 
 The **RTO** can be considered for the **failover** and **failback** procedures, both resulting in a functional state.
 
-- **failover** has an **RTO** of **15-20** minutes to restore a functional state, excluding DNS considerations.
-- **failback** has an **RTO** of **25-30 + X** minutes to restore a functional state. Where X is the time it takes to back up and restore Elasticsearch, which is highly dependent on the setup and chosen [Elasticsearch backup type](https://www.elastic.co/guide/en/elasticsearch/reference/current/snapshots-register-repository.html#ess-repo-types).
+- **failover** has an **RTO** of **< 1** minute to restore a functional state, excluding DNS considerations.
+- **failback** has an **RTO** of **5 + X** minutes to restore a functional state. Where X is the time it takes to back up and restore Elasticsearch, which is highly dependent on the setup and chosen [Elasticsearch backup type](https://www.elastic.co/guide/en/elasticsearch/reference/current/snapshots-register-repository.html#ess-repo-types).
```

Comment:
:question: What is the reason for 5m?

Reply:
I took the estimate from our automated tests for the procedure and comes mostly from installing C8.
First install of C8 and waiting for Elastic to be ready is roughly 2-3 minutes.
Later installing Operate / Tasklist is another minute.
In-between are the steps that need to be taken when automated of roughly a minute. (It's automated, so a manual execution will even take longer).

Reply:
Since RTO contains the time to recover as well, it's roughly 5 minutes from my side.
I'm open to other suggestions if you think it should be more or less.

Reply:
Thanks. I'm wondering if we have to specify the 5 minutes. Can we say it in general that it depends on the ES backup and restore + some additional time that needs to install and start C8 components in the new region? 5 minutes seems arbitrary and can also depends on whether it is manual, automated etc.

Reply:
I get your point about the arbitrary number, but we had before 25-30 minutes.
Just saying it takes whatever time for Backup and whatever time for Install sounds very fuzzy and questionable on why it can't be pinpoint.
I mean Elastic + WebApps have health checks in place, that's why there is a guaranteed lower bound anyway.
Sure there are more factors that can play in but I feel like giving 0 guidance is even less useful.

How about we rephase it to:

> - **failback** has an **RTO** of **5 + X** minutes to restore a functional state. Where X is the time it takes to back up and restore Elasticsearch, which is highly dependent on the setup and chosen [Elasticsearch backup type](https://www.elastic.co/guide/en/elasticsearch/reference/current/snapshots-register-repository.html#ess-repo-types).
During our automated tests, the reinstallation and reconfiguration of Camunda 8 takes 5 minutes. This can serve as a general guideline for the time required, though your experience may vary depending on your available resources and familiarity with the operational procedure.

Reply:
The rephrase sounds good to me. :+1: